### DISCUSSION TITLE
OncogenicPathway

### DISCUSSION THREAD

**Discussion by avrajit** [PoisonAlien/maftools#1060] [Q&A]
How I will get genes name from oncogenicPathway? As per the maual I did this
`pwa<- pathways(maf=maf, pathdb = 'smgbp', plotType = 'bar')`
`plotPathways(maf = maf, pathlist = pwa)`

It is showing correct plot and informations but how to find out which  genes name enriched under each pathway terms?
please help

**Comment by PoisonAlien** [comment]
Hi,

You could try the below

```r
pwa <- pathways(maf=maf, pathdb = 'smgbp', plotType = 'bar')
attr(pwa, "genes")
```

Although it returns all the members of the pathway (not enriched).
